docs/2_5_fmu_distribution.adoc

```diff
@@ -50,7 +50,8 @@ sources                             // directory containing the C sources (optio
 binaries                            // directory containing the binaries (optional)
    x86_64-windows                   // binaries for Windows on Intel 64-bit
       <modelIdentifier>.dll         // shared library of the FMI implementation
-      <other DLLs>                  // the DLL can include other DLLs
+      <other files>                 // other platform dependend files 
```

Comment:
@chrbertsch is it clear enough that <other files> are allowed in parallel subfolders as well?